types/web3/web3-tests.ts

```diff
@@ -28,7 +29,9 @@ const sendSignedTransactionTxReceipt0: PromiEvent<TransactionReceipt> = web3.eth
 const sendSignedTransactionTxReceipt1: PromiEvent<TransactionReceipt> = web3.eth.sendSignedTransaction("")
     .on("transactionHash", (txHash: string) => { });
 const sendSignedTransactionTxReceipt2: PromiEvent<TransactionReceipt> = web3.eth.sendSignedTransaction("")
-    .on("receipt", (txReceipt: TransactionReceipt) => { });
+    .on("receipt", (txReceipt: TransactionReceipt) => {
+       assert.ok('boolean' === typeof txReceipt.status);
```

Comment:
That is not a valid test. This code is never run, but just compiled. So no need for assert. Please read the DT [docs on this](https://github.com/DefinitelyTyped/DefinitelyTyped#create-a-new-package). 
```suggestion
       const { status } : {status: boolean}  = txReceipt; 
```
TS would fail to compile if the return value would still be a string.

Reply:
Also remove the import of `assert` then.

Reply:
Thank you! Done!